SwiftCheck/Property.swift

```diff
@@ -254,12 +276,25 @@ public enum Callback {
 
 /// The type of callbacks SwiftCheck can dispatch.
 public enum CallbackKind {
-	///
+	/// The callback is invoked after a counterexample is displayed.
 	case Counterexample
+	/// The callback is invoked after every test.
 	case NotCounterexample
 }
 
-public enum TestResultMatcher {
+/// The types of quantification SwiftCheck can perform.
+public enum Quantification {
+	/// Universal Quantification ("for all").
+	case Universal
+	/// Existential Quanfication ("there exists").
+	case Existential
+	/// Uniqueness Quantification ("there exists one and only one")
+	// case Uniqueness
+	/// Counting Quantification ("there exist exactly k")
+	// case Counting
```

Comment:
If you do add these, I suggest you just do `Counting`. I don’t think `Uniqueness` adds much to the interface and, presumably the implementation of `Counting` would subsume `Uniqueness`.